target/maven-archiver/pom.properties

```diff
@@ -1,3 +1,7 @@
 artifactId=E-commerceFinalProject
+<<<<<<< HEAD
+groupId=com.
+=======
 groupId=com.example
+>>>>>>> main
```

Comment:
there are some conflicts we need to resolve. please be careful as this is very dangerous when conflicts remain in code.

Reply:
Musie kindly resolved all conflicts